source/client/service_impl.cc

```diff
@@ -33,11 +33,7 @@ void ServiceImpl::handleExecutionRequest(const nighthawk::client::ExecutionReque
     return;
   }
 
-  ProcessImpl process(*options, time_system_);
```

Comment:
Note to reviewers: the fix is to not do this here. We do this once now at construction time, and hold on to the logging context throughout the lifetime of the `ServiceImpl`.
Note that `ProcessImpl` instantiations will change the log level per inbound configured options. We only allow one `ProcessImpl` instantiation at a time today, so this might be allright for now.